### PR TITLE
add manifest for marketplace

### PR DESCRIPTION
**Pull request by jcantrill**
This PR adds manifest for marketplace

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @bparees

File: controller-manifests/elasticsearch-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+          - servicemonitors
+          verbs:
+          - "*"
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterroles
+          - clusterrolebindings
+          verbs:
+          - "*"
```
Comment (ewolinetz) [inline controller-manifests/elasticsearch-operator.v0.0.1.clusterserviceversion.yaml]: missing a latest change here... 
```
- nonResourceURLs:
  - "/metrics"
  verbs:
  - get
- apiGroups:
  - authentication.k8s.io
  resources:
  - tokenreviews
  - subjectaccessreviews
  verbs:
  - create
- apiGroups:
  - authorization.k8s.io
  resources:
  - subjectaccessreviews
  verbs:
  - create
```

**Comment by bparees** [comment]
comments from https://github.com/openshift/cluster-logging-operator/pull/113 apply here as well
/hold

**Comment by jcantrill** [comment]
@bparees updated to 4.1.0

**Comment by smarterclayton** [comment]
This PR never passed e2e-aws, why did you merge it?

**Comment by smarterclayton** [comment]
Please revert this, it broke all CI across the org because you merged with broken e2e-aws.

**Pull request merged**